packages/extension-browser/src/devtools/views/controls/summary.css

```diff
@@ -1,7 +1,6 @@
 .root {
   cursor: pointer;
   line-height: 1.5rem; /* 24px */
-  width: max-content;
```

Comment:
Oh... Let me check if hit area is being increased to empty space. I removed it because it was causing unnecessary overflow in small screen. Because of which two scroll bars were getting shown at the bottom.
Removing it enables word wrapping for hint name.

Reply:
You are right, hit area extend to full width.

Reply:
`width: fit-content` serves both purpose. Hint name wraps in smaller screen and hit area also remain limited to content width only. I will test it now in case it creates some other issue. and will update soon.

Reply:
Sounds like that might do the trick - curious to hear how the testing goes.

We've been through a few iterations to get this one fine-tuned how we like it. For reference, here was when we introduced `max-content` to fix a different bug: https://github.com/webhintio/hint/commit/04ff44e194cd102b221b4b4642f99e3a204340e0

Reply:
It seems to be more appropriate than `max-content`. It uses `max-content` only if enough space is available, otherwise it switches to auto. However `max-content` always extend the content width to its size.
So it was causing overflow.
Please refer its docs also. I am trying to understand the difference correctly.
https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content

Reply:
I agree `fit-content` seems more appropriate.